config.sh

```diff
@@ -113,7 +113,6 @@ check_pkg "sqlite3" || fail "sqlite3"
 check_pkg "libcurl" || check_custom "libcurl" "curl-config" || fail "libcurl"
 check_pkg "libxml-2.0" || check_custom "libxml2" "xml2-config" || fail "libxml2"
 check_pkg "stfl" || fail "stfl"
-check_cmd "asciidoctor" || fail "asciidoctor"
```

Comment:
Would it make sense to show a warning here if asciidoctor is not found?
For example:
`check_cmd "asciidoctor" || echo "install asciidoctor if you want to build the documentation"`

Reply:
Good point, fixed.

Reply:
Nice, looks good to me